docs/configuration.md

```diff
@@ -942,6 +942,210 @@ editing the `conf` file in a text editor. Use the examples as reference.
+        <td colspan="2">No value</td>
+    </tr>
+    <tr>
+        <td>Example</td>
+        <td colspan="2">@code{}
+            dd_manual_resolution = 1920x1080
+            @endcode</td>
+    </tr>
+</table>
+
+### dd_refresh_rate_option
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            Perform additional refresh rate configuration for the display device:
+            <ul>
+                <li>`disabled` - perform no additional configuration.</li>
+                <li>`automatic` - change refresh rate to the requested FPS value from the client.</li>
+                <li>`manual` - change refresh rate to the user specified one (set via `dd_manual_refresh_rate`).</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td colspan="2">@code{}automatic@endcode</td>
+    </tr>
+    <tr>
+        <td>Example</td>
+        <td colspan="2">@code{}
+            dd_refresh_rate_option = manual
+            @endcode</td>
+    </tr>
+</table>
+
+### dd_manual_refresh_rate
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            @note{`dd_refresh_rate_option` must be set to `manual`}
+            Specify manual refresh rate to be used.
+        </td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td colspan="2">No value</td>
+    </tr>
+    <tr>
+        <td>Example</td>
+        <td colspan="2">@code{}
+            dd_manual_resolution = 120
+            dd_manual_resolution = 59.95
+            @endcode</td>
+    </tr>
+</table>
+
+### dd_hdr_option
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            Perform additional HDR configuration for the display device:
+            <ul>
+                <li>`disabled` - perform no additional configuration.</li>
+                <li>`automatic` - change HDR to the requested state from the client if the display supports it.</li>
+            </ul>
+        </td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td colspan="2">@code{}automatic@endcode</td>
+    </tr>
+    <tr>
+        <td>Example</td>
+        <td colspan="2">@code{}
+            dd_hdr_option = disabled
+            @endcode</td>
+    </tr>
+</table>
+
+### dd_wa_hdr_toggle
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            @note{This option works independently of `dd_hdr_option`}
+            When using virtual display device as for streaming, it might display incorrect (high-contrast) color.
+            With this option enabled, Sunshine will try to mitigate this issue.
+        </td>
+    </tr>
+    <tr>
+        <td>Default</td>
+        <td colspan="2">@code{}false@endcode</td>
+    </tr>
+    <tr>
+        <td>Example</td>
+        <td colspan="2">@code{}
+            dd_wa_hdr_toggle = true
```

Comment:
```suggestion
            dd_wa_hdr_toggle = enabled
```

Reply:
This is not correct:
![image](https://github.com/user-attachments/assets/d0728adc-ca3f-4a47-ab76-b3bb4f3c8fe5)

The saved option is boolean.

Reply:
> I really like the idea of check boxes, but currently we aren't using those anywhere else. So I think that should come in a separate PR changing them universally from the select menus.

I have fixed this issue already. The `enabled` works with them like a charm. I hope that I don't need to change the checkbox to the menu (your remark leaves it open for interpretation)?

Reply:
It would be quite weird to have one checkbox in the ui, while everywhere else is a select menu.

I know you don't like to open more than one PR at once, but changing them out globally would be better. If a separate PR is created to change that pattern, then it's fine to use it here as well.

Reply:
I'll create a separate PR then

Reply:
I'll prioritize reviewing and merging it whenever it's ready. Thanks!